<launch>
  <arg name="mav_name" default="hummingbird"/>
  <arg name="world_name" default="demo_gap"/>
  <arg name="world_camera" default="false"/>

  <arg name="map_file" default="$(find nav_quadrotor)/maps/demo_world.yaml"/>

  <include file="$(find nav_quadrotor)/launch/gazebo_empty.launch">
    <arg name="world_name" value="$(arg world_name)"/>
    <arg name="map_file" value="$(arg map_file)"/>
  </include>

  <include file="$(find nav_quadrotor)/launch/gazebo_camera.launch" if="$(arg world_camera)">
    <arg name="model_file" value="$(find nav_quadrotor)/urdf/gazebo_cameras/barrier_camera.sdf"/>
    <arg name="roll" value="0"/>
    <arg name="yaw" value="-3.14159"/>
    <arg name="pitch" value="1.15"/>
    <arg name="x" value="5.5"/>
    <arg name="y" value="-2.214876"/>
    <arg name="z" value="8.1"/>
  </include>

</launch>
